### PR TITLE
Docusaurus version

### PR DESCRIPTION
**Pull request by tanishqr-ck**
## Related Issue

- Information about the related issue
This is the docusaurus version of devpath
Closes: #37 

### Describe the changes you've made
I have made a docusaurus version of this devpath for better readability.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by tanishqr-ck** [comment]
Here is the preview [doc version](https://docs-sandy-one.vercel.app/)

File: docs/intro.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+sidebar_position: 1
+
+title: 🤝🏻 Introduction
+---
+
+# Introduction to Blockchain Developer Path
+
+
+
+Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of web3community. We gather content about Blockchain developers. You don't need to see all the resources, but choose what content is best for you.<br />
```
Comment (shuklaritvik06) [inline docs/intro.md:11]:
```suggestion
Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of WebXDAO. We gather content about Blockchain developers. You don't need to see all the resources, but choose what content is best for you.<br />
```

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]

**Review by shuklaritvik06 — APPROVED** [review]
LGTM, Awesome work! 🔥

File: docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            ],
+            docId: 'comm',
+            position: 'left',
+            label: 'Community',
+            
+          },
+          
+          {
+            label: "Join Us",
+            href:'https://github.com/WebXDAO/start-here/issues',
```
Comment (shuklaritvik06) [inline docusaurus.config.js:82]:
```suggestion
            href:'https://github.com/WebXDAO/start-here/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community',
```

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]

**Comment by AbhinavMV** [comment]
Awesome work. 🔥
just one suggestion - in Free Training section there is just one line. maybe a link to github repo can be added(like in the readme file). or something else can be added to make it more useful.

**Review by vinzvinci — COMMENTED** [review]
Looks good, but I consider Abhinav's suggestion ☝️

**Review by KukretiShubham — APPROVED** [review]
LGTM, I agree to abhinav's suggestion

**Comment by vinzvinci** [comment]
Hi @tanishqr-ck, is there anything you need help with?

**Comment by mkubdev** [comment]
Awesome work @tanishqr-ck 💫! Merge ok ? Maybe @AbhinavMV suggestion can be done later

**Review by mkubdev — APPROVED** [review]
LGTM, let's ship it 💥

**Comment by vinzvinci** [comment]
Thank you for merging and right decision. Congrats 👏

**Pull request merged**